### ISSUE TITLE
Can't edit break tipps

### ISSUE THREAD

**Issue by Katzinski** [hovancik/stretchly#663]
### Prerequisites

- [X] I'm using latest version: v 1.00 (DEB 1.0.0-1933)
- [X] I've checked [Known issues](https://github.com/hovancik/stretchly#known-issues)
- [X] I've checked [Open issues](https://github.com/hovancik/stretchly/issues)

### Description

In v 0.21.1, I could edit the _config.json_ file to change break ideas (toggle display from true to false and edit break tipp). In v 1.0.0, the same break tipps are displayed no matter the edits I do. Other edits to the _config.json_ like changing pause duration do take effect.

Operating system: Xubuntu 18.04.5 LTS

### Steps to Reproduce

1. Install Stretchly for the first time, start it and quit it
2. Go to ~/.config/Stretchly (watch capital "S" - "Stretchly", not "stretchly")
3. Change code of all breaks and minibreaks but one from _enabled: true_ to _enabled: false_ and add "REMARK" to the break text
4. Start Stretchly and jump to next break 
5. Default break tip will show up - instead of the one set to _true_ and containing "REMARK"
6. Jump to the next break
7. Next default break tip will show up - instead of the only one set to _true_ containing "REMARK" 

**Expected behavior:** 
please see above step 5 and 7

**Actual behavior:** 
please see above step 5 and 7

**Reproduces how often:** 
Always

### Additional Information

none

**Comment by Katzinski** [comment]
I can't remember if I did _Restore Defaults_ at some point but I'd think that the app is restarted every then and now with system starts. So that would not explain what I had.

I read your notice on name change when I downloaded v 1.0 and copied the texts from the old to the new config file but I failed to remove/rename the old folder in ~/.config before I launched v 1.0 ... 

Anyway, it works for me. Thank you so much for your help and for all your efforts for this valuable app!
